libs/ml/tests/unit/model/dnn_classifier_test.cpp

```diff
@@ -120,7 +120,7 @@ TYPED_TEST(DNNClassifierModelTest, adagrad_dnnclasifier)
 {
   using DataType = typename TypeParam::Type;
   ASSERT_TRUE(classifier_details::RunTest<TypeParam>(fetch::ml::OptimiserType::ADAGRAD,
-                                                     DataType{1e-2f}, DataType{0.01f}, 400));
+                                                     DataType{1e-2f}, DataType{0.03f}, 400));
 }
```

Comment:
scientific notation

Reply:
and string construction

Reply:
ML tests will be defloated in a separate PR